### DISCUSSION TITLE
en/chapter_data_structure/character_encoding/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1519] [Announcements]
# en/chapter_data_structure/character_encoding/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_data_structure/character_encoding/

**Comment by isomoes** [comment]
As you mentioned, there is a "difference in how strings are stored in files or transmitted over networks". Therefore, it is necessary to discuss Base64 encoding as an example of character encoding that is effective for network transmission ?🎃
